parachain/pallets/omni-account/src/identity_context.rs

```diff
@@ -0,0 +1,51 @@
+use crate::{BlockNumberFor, Config, Web3Network};
+use parity_scale_codec::{Decode, Encode};
+use scale_info::TypeInfo;
+use sp_std::vec::Vec;
+
+#[derive(Clone, Eq, PartialEq, Default, Debug, Encode, Decode, TypeInfo)]
+pub enum IdentityStatus {
```

Comment:
Maybe we don't need this status either - it's been rarely/never used

It could be useful to prevent some member identity from accessing the OmniAccount in contingency (e.g. when I know some account is compromised) - but the hacker could have updated this setting using the compromised account

Or we leave as it is hoping the hacker doesn't know about it and OmniAccount owner might have time window to take actions
Wdyt?

Reply:
I think it could be a nice feature, we could make it similar to a Multisig to only allow re-activating an identity if at least two members (or more) of the OmniAccount approve it or something like this. (The same for removing an identity if we want to support this at some point)

Reply:
But if we don't want to support this, or at least now and we are removing the web3networks, I think we can get rid of the identity context all together, I think the link_block was used to sort the IDGraph but now we don't need to sort it

Reply:
Multisig-ish control is a good idea.

Yes it seems we don't need this Context at the moment - the `link_block` was designed for potential future feature like if we want to ask users to "renew" the ownership in X blocks.